discord-bot-demo/backend/supabase/config.toml

```diff
@@ -0,0 +1,60 @@
+# A string used to distinguish different Supabase projects on the same host. Defaults to the working
+# directory name when running `supabase init`.
+project_id = "viewer-polls-supabase-example"
```

Comment:
I'd update the project_id to discord-bot-example

Reply:
impl in https://github.com/trufflehq/truffle-packages/pull/8/commits/b02c18d62d112b4eed2c6f29ab494c36832ad2f2